src/utilities/OpenStudioApplicationPathHelpers.cxx.in

```diff
@@ -49,6 +49,10 @@
 
 namespace openstudio {
 
+std::string getOpenStudioApplicationVersion() {
+  return "${OPENSTUDIOAPPLICATION_VERSION}";
```

Comment:
@jmarrec is this the right version to use?  does it include pre-release info?

Reply:
https://github.com/openstudiocoalition/OpenStudioApplication/blob/ce243ace4aa158f1ca9dad3b70615c366b51f3a9/CMakeLists.txt#L169-L182

Reply:
As long as you do comparison via VersionString it should be fine as it'll handle the case of wether you do have a pre-release or not:

https://github.com/NREL/OpenStudio/blob/ff0e898ff96e0653e1b72da4200a3403b927fdaa/src/utilities/core/Compare.cpp#L249-L262